test/Microsoft.Azure.Webjobs.Extensions.Storage.UnitTests/HostCallTests.cs

```diff
@@ -1726,5 +1655,73 @@ private struct StructTableEntity
         {
             public string Value { get; set; }
         }
+
+        internal class CustomBlobConverterExtensionConfigProvider : IExtensionConfigProvider
+        {
+            public void Initialize(ExtensionConfigContext context)
```

Comment:
Registering custom Blob conversions via an extension.

Reply:
Yes, I agree. Was going to ask if you'd moved things to use converters when you removed the CloudBlobStreamBinder (as it was hard to tell due to the reshuffling above). But I think this is a good step to get there.

It also seems like we could support POCO with a converter that we auto-register with the Blob extension.